### PR TITLE
Update to 1.2.6

### PR DESCRIPTION
**Pull request by davidh-ssec**

### CODE REVIEW

**Comment by davidh-ssec** [comment]
Guess I'll wait for the DNS issues going around to be resolved...

**Comment by ocefpaf** [comment]
Closing and re-opening to see if we can start AppVeyor.

**Comment by ocefpaf** [comment]
@davidh-ssec can you rerender this recipe with `conda-smithy`:

- install conda-smithy: `conda install -c conda-forge conda-smithy` in your root env.
- type: `conda smithy rerender` at the root directory of the feedstock
- commit the changes.

That should fix the AppVeyor failures.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,10 +7,10 @@ package:
 source:
     fn: pyresample-{{ version }}.tar.gz
     url: https://github.com/pytroll/pyresample/archive/v{{ version }}.tar.gz
-    sha256: b4c8be42f2423eb50ab0345054d4ade7c7b275c250b826433817aba014a537f5
+    sha256: 8a0da50582358933d3fd0c88ca2cad1ba572dc19fc4474b0c3ccaed448dee768
 
 build:
-    number: 0
+    number: 1
```
Comment (davidh-ssec) [inline recipe/meta.yaml]: ...and crap I forgot to fix the build number.

**Comment by ocefpaf** [comment]
Thanks @davidh-ssec!

**Pull request merged**